testsuite/tests/contrib/lkt_semantic/property_error/test.lkt

```diff
@@ -1,5 +1,7 @@
 fun prop1(): Bool = raise PropertyError("error")
 fun prop2(): Bool = (raise PropertyError("error"))
+fun prop21(): Bool = raise (PropertyError("error"))
+fun prop22(): Bool = raise (((PropertyError("error"))))
```

Comment:
OK, it needs a bit more of work then! I'm on it.

Reply:
@pmderodat @thvnx I think Laurent can do that in a separate PR. In general let's try to keep PRs as small as possible since they have a tendency to linger (ie. not add new cases to existing functional PRs)

Reply:
I've just added a TODO for that.